share/ocr-languages.json

```diff
@@ -1,66 +1,130 @@
 {
-    "Afrikaans": "arf",
+    "Afrikaans": "afr",
+    "Amharic": "amh",
     "Arabic": "ara",
     "Assamese": "asm",
     "Azerbaijani": "aze",
+    "Azerbaijani (Cyrillic)": "aze_cyrl",
     "Belarusian": "bel",
     "Bengali": "ben",
+    "Tibetan Standard": "bod",
+    "Bosnian": "bos",
+    "Breton": "bre",
     "Bulgarian": "bul",
     "Catalan": "cat",
+    "Cebuano": "ceb",
     "Czech": "ces",
     "Chinese - Simplified": "chi_sim",
+    "Chinese - Simplified (vertical)": "chi_sim_vert",
     "Chinese - Traditional": "chi_tra",
+    "Chinese - Traditional (vertical)": "chi_tra_vert",
     "Cherokee": "chr",
+    "Corsican": "cos",
+    "Welsh": "cym",
     "Danish": "dan",
+    "Danish - Fraktur": "dan_frak",
     "German": "deu",
+    "German - Fraktur": "deu_frak",
+    "Divehi": "div",
+    "Dzongkha": "dzo",
     "Greek": "ell",
     "English": "eng",
     "English, Middle (1100-1500)": "enm",
     "Esperanto": "epo",
+    "Math / equation detection module": "equ",
```

Comment:
I think this is a module that this is module rather than a language. So I'd say it should be omitted.

Reply:
Done in 7a4e244a3071fb0079811fcfdfe8e2c66e8e6dd5.